Formula/zli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class Zli < Formula
+  desc "BastionZero cli"
+  homepage "https://www.bastionzero.com"
+  url "https://github.com/bastionzero/zli/releases/download/4.9.0/zli-4.9.0.tar.gz"
+  sha256 "553b7a82a1b4b5015acc9bbe645afcca1fb351e92dbfe6514d7355e8df145136"
+  license "Apache-2.0"
+  head "https://github.com/bastionzero/zli.git"
+
+  bottle :unneeded
```

Comment:
Why is this `bottle :unneeded`?

Reply:
Not sure, I think this is an artifact of me learning how to write a brew formula. Will remove.